### PR TITLE
june ux blog

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-blog-202106](https://developer.blackbaud.com/skyux?_sp.spa=pr-blog-202106)

File: src/app/blog/index.html [hunk 1]
```diff
@@ -4,6 +4,21 @@
...
+	<article class="blog-post">
+		<header>
+			<stache-page-anchor>
+				Design update — June 2021
+			</stache-page-anchor>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+			<h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-lookup/docs/lookup">Lookup enhancements</a></h4>
+				<p>We added functionality to the lookup component that allows users to add new items on the fly and browse the search matches. With these enhancements we will be deprecating <a href="https://developer.blackbaud.com/skyux-lookup/docs/autocomplete">autocomplete</a> and <a href="https://developer.blackbaud.com/skyux-select-field/docs/select-field">select field</a>; any use of those should be replaced with lookup.</p>
```
Comment (blackbaud-johnly) [inline src/app/blog/index.html:16]:
```suggestion
				<p>We updated the lookup component to allow users to add items on the fly and to open a picker that displays options. With these enhancements, we will deprecate <a href="https://developer.blackbaud.com/skyux-lookup/docs/autocomplete">autocomplete</a> and <a href="https://developer.blackbaud.com/skyux-select-field/docs/select-field">select field</a>; lookup should replace any use of those components.</p>
```

File: src/app/blog/index.html [hunk 2]
```diff
@@ -4,6 +4,21 @@
...
+			<h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-text-editor/docs/text-editor">Text editor</a></h4>
+				<p>We added a text editor that allows users to enter formatted text.</p>
```
Comment (blackbaud-johnly) [inline src/app/blog/index.html:18]: Should we mention that this is a beta release? Or just rely on the note on the component docs?

**Review by blackbaud-johnly — APPROVED** [review]
I suggested some tweaks to the first entry and asked a question about the second entry, but I'm approving the PR since I don't think either are dealbreakers.

File: src/app/blog/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/blog/index.html:18]: Ran this question by @Blackbaud-KerryCampbell, and we're going to rely on the note in the component docs.

**Review by blackbaud-johnly — APPROVED** [review]

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-blog-202106](https://developer.blackbaud.com/skyux?_sp.spa=pr-blog-202106)

**Pull request merged**